.travis.yml

```diff
@@ -25,16 +26,19 @@ addons:
 before_install:
   - gem update --system
   - npm i nsp -g
+  - npm i snyk -g
 before_script:
   - mongo --version
   - mongod --version
+  - snyk test
+# adding influxdb repository
```

Comment:
Should be possible to catch the calls and check the correct points are passed, we do it for agenda jobs and emails in https://github.com/Trustroots/trustroots/blob/master/testutils.js.